_data/utilities.yml

```
@@ -87,7 +87,8 @@ websites:
       - phone
       - totp
       - proprietary
-    doc: https://www.coned.com/en/accounts-billing/new-my-account-faqs
+    doc: https://www.coned.com/-/media/files/coned/documents/small-medium-large-businesses/building-project-center/contractor-resources/user-registration-guide.pdf?la=en
+    exception: "Requires phone # to set up. Can be bypassed by customer support." 
```

Proper English without slang is preferred when writing exceptions. That way more people and translation services can understand it.
```suggestion
    exception: "Requires phone number to set up. Can be bypassed by customer support." 
```